src/monitor/node_metadata.c

```diff
@@ -158,7 +158,7 @@ TupleToAutoFailoverNode(TupleDesc tupleDescriptor, HeapTuple heapTuple)
 	pgAutoFailoverNode->pgsrSyncState =
 		SyncStateFromString(TextDatumGetCString(pgsrSyncState));
 	pgAutoFailoverNode->reportTime = DatumGetTimestampTz(reportTime);
-	pgAutoFailoverNode->walDelta = DatumGetInt64(walDelta);
+	pgAutoFailoverNode->currentLSN = DatumGetInt64(latestLSN);
```

Comment:
This is confusing, isn't it?

We should stick to the current norm for naming things. On the client side, `pg_autoctl` sees the CURRENT version of things, then sends it to the monitor, for storage and processing. The monitor might use the value immediately, or later from another's node request, at which point we know that the value is the latest reported value rather than the current one. So in the monitor we accept the CURRENT value and then register it as the REPORTED value. 

I don't think we use LATEST at the moment in our variable names, to avoid the confusion.